src/data/common/Dhis2DataStoreDataForm.ts

```diff
@@ -270,6 +282,7 @@ const DataStoreConfigCodec = Codec.interface({
                         type: oneOf([exactly("dataElement"), exactly("dataElementExternal")]),
                         code: string,
                         condition: optional(string),
+                        disabled: optional(string),
```

Comment:
I think this was fixed on another branch, right? https://github.com/EyeSeeTea/d2-autogen-forms/pull/81/files

Can you check the PR and update this PR if necessary.